docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1748 @@
+
+.. |qiskit_aer.primitives| replace:: ``qiskit_aer.primitives``
+.. _qiskit_aer.primitives: https://qiskit.org/documentation/locale/de_DE/apidoc/aer_primitives.html
+
+.. |qiskit_aer.primitives.Sampler| replace:: ``qiskit_aer.primitives.Sampler``
+.. _qiskit_aer.primitives.Sampler: https://qiskit.org/documentation/locale/de_DE/stubs/qiskit_aer.primitives.Sampler.html
+
+.. |qiskit_aer.primitives.Estimator| replace:: ``qiskit_aer.primitives.Estimator``
+.. _qiskit_aer.primitives.Estimator: https://qiskit.org/documentation/locale/de_DE/stubs/qiskit_aer.primitives.Estimator.html
+
+.. |qiskit_ibm_runtime| replace:: ``qiskit_ibm_runtime``
+.. _qiskit_ibm_runtime: https://qiskit.org/documentation/partners/qiskit_ibm_runtime/primitives.html
+
+.. |qiskit_ibm_runtime.Sampler| replace:: ``qiskit_ibm_runtime.Sampler``
+.. _qiskit_ibm_runtime.Sampler: https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.Sampler.html
+
+.. |qiskit_ibm_runtime.Estimator| replace:: ``qiskit_ibm_runtime.Estimator``
+.. _qiskit_ibm_runtime.Estimator: https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.Estimator.html
+
+..  attention::
+
+    Most references to the :class:`qiskit.primitives.Sampler` or :class:`qiskit.primitives.Estimator` in this guide
+    can be replaced with instances of the:
+
+    - Aer primitives (|qiskit_aer.primitives.Sampler|_, |qiskit_aer.primitives.Estimator|_)
+    - Runtime primitives (|qiskit_ibm_runtime.Sampler|_, |qiskit_ibm_runtime.Estimator|_ )
+    - Terra backend primitives (:class:`qiskit.primitives.BackendSampler`, :class:`qiskit.primitives.BackendEstimator`)
+
+    Certain classes, such as the
+    :class:`~qiskit.opflow.expectations.AerPauliExpectation`, can only be replaced by a specific primitive instance
+    (in this case, |qiskit_aer.primitives.Estimator|_ ), or require a specific option configuration.
+    If this is the case, it will be explicitly indicated in the corresponding section.
+
+Contents
+========
+
+This document covers the migration from these opflow sub-modules:
+
+**Operators**
+
+- `Operator Base Class`_
+- `Operator Globals`_
+- `Primitive and List Ops`_
+- `State Functions`_
+
+**Converters**
+
+- `Converters`_
+- `Evolutions`_
+- `Expectations`_
+
+**Gradients**
+
+- `Gradients`_
+
+
+Operator Base Class
+===================
+*Back to* `Contents`_
+
+.. |qiskit.quantum_info.BaseOperator| replace:: ``qiskit.quantum_info.BaseOperator``
+.. _qiskit.quantum_info.BaseOperator: https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/quantum_info/operators/base_operator.py
+
+The :class:`qiskit.opflow.OperatorBase` abstract class can be replaced with |qiskit.quantum_info.BaseOperator|_ ,
+keeping in mind that |qiskit.quantum_info.BaseOperator|_ is more generic than its opflow counterpart.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+   * - :class:`qiskit.opflow.OperatorBase`
+     - |qiskit.quantum_info.BaseOperator|_
+
```

Comment:
The rendering of list table for the theme is not ideal. The width changes depending on the content. Have you tried using other kinds of tables in Sphinx such as a normal `table`? https://docutils.sourceforge.io/docs/ref/rst/directives.html#toc-entry-21

I can totally see that list table is much easier to right, especially for this case where the entries of each cell is short. Perhaps can we improve the rendering in the qiskit sphinx theme.

<img width="864" alt="Screenshot 2023-03-01 at 12 05 55" src="https://user-images.githubusercontent.com/7631333/222122083-2b4e37b3-ece3-4586-8936-77c7dfb2aa05.png">